projects/pastanaga/src/lib/richtext/richtext.component.scss

```diff
@@ -0,0 +1,8 @@
+::ng-deep {
+    @import "../../../../../../../../node_modules/medium-editor/src/sass/medium-editor";
+    @import "../../../../../../../../node_modules/medium-editor/src/sass/themes/default";
+}
```

Comment:
I'm a little scared of this `ng-deep`. Is it really necessary? 
By the way there is a [easier way](https://stackoverflow.com/a/49523565/4912036) to import styles from `node_modules` using `~`